Data/Graph/Inductive/Query/Cycles.hs

```diff
@@ -100,45 +96,41 @@ cyclesFor n st0 =
                filter (\c -> n `gelem` c) $
                cisComponents st0
   in if noNodes n_comp > 1
```

Comment:
Note that `noNodes` is `O(|V|)` for PatriciaTree (but `O(1)` for `Tree`); it would probably be better to do something like `not . null . drop 1 . nodes $ n_comp` which should be `O(1)`.

Reply:
Ah, didn't know that. Fixed.